packages/delegation-process/.env

```
@@ -18,7 +18,7 @@ READMODEL_DB_PASSWORD=example
 READMODEL_DB_PORT=27017
 
 WELL_KNOWN_URLS="http://127.0.0.1:4500/jwks.json"
-ACCEPTED_AUDIENCES="dev.interop.pagopa.it/ui,dev.interop.pagopa.it/ui,dev.interop.pagopa.it/internal,dev.interop.pagopa.it/m2m,dev.interop.pagopa.it/m2m"
+ACCEPTED_AUDIENCES="dev.interop.pagopa.it/ui,dev.interop.pagopa.it/internal,dev.interop.pagopa.it/m2m,dev.interop.pagopa.it/m2m-admin"
```

No need to create a new audience
```suggestion
ACCEPTED_AUDIENCES="dev.interop.pagopa.it/ui,dev.interop.pagopa.it/internal,dev.interop.pagopa.it/m2m,dev.interop.pagopa.it/m2m"
```